engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -201,6 +223,7 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
 
   val iOExecutionContext = context.system.dispatchers.lookup("akka.dispatchers.io-dispatcher")
   implicit val ec = context.dispatcher
+  implicit val timeout = Timeout(5.seconds)
```

Comment:
Question for reviewers - What should the value for timeout be?

Reply:
Since this is a non-synchronous activity, I would lean higher here (like maybe 60 seconds)? But not a strong opinion and happy to be talked down from that

Reply:
TOL can we provide this value explicitly where it's used? Or rename it something less generic? In the context of the materialize workflow actor, a global value called "timeout" to refer specifically to github auth lookup is a little odd

Reply:
This is coming from the `GithubAuthVendingSupport` trait. I am not sure how to provide this value where it is used since its coming from the trait? 🤔